src/porepy/viz/exporter.py

```diff
@@ -343,24 +344,22 @@ def import_state_from_vtu(
         from deepdiff import DeepDiff
 
         # Aux. method: Inverse of _to_vector_format, used to define vector-ranged values
-        def _from_vector_format(
-            value: np.ndarray, grid: Union[pp.Grid, pp.MortarGrid]
-        ) -> np.ndarray:
+        def _from_vector_format(value: np.ndarray, num: int) -> np.ndarray:
```

Comment:
I suggest a better name for `num`, e.g., `ndofs`, `num_dofs`, or `size`.

Reply:
I agree, I'll use num_dofs